lib/docker-image-tree.coffee

```diff
@@ -32,4 +33,6 @@ exports.annotateTree = annotateTree = (layer_mtimes, tree) ->
 	}
 
 exports.dockerImageTree = dockerImageTree = ->
-	docker.listImagesAsync(all: true).then(createTree)
+	dockerUtils.getDocker()
+	.then (docker) ->
```

Comment:
You could change this to
```coffee
.call('listImages', all: true)
.then(createTree)
```
\- at the least unindenting the `.then(createTree)` would be nice